frontend/src/pages/create_campaign/index.js

```diff
@@ -66,102 +114,77 @@ const CreateCampaign = () => {
     } else {
       console.log("something fucked up");
     }
+    */
   };
-
-  const onFileUpload = async (acceptedFiles) => {
-    const fileUrl = await fileToDataUrl(acceptedFiles[0]);
-    setCover(fileUrl);
-  };
-
   return (
-    <CampaignContainer>
-      <Dropzone
-        onDrop={acceptedFiles => onFileUpload(acceptedFiles)}
-        accept={["image/jpeg", "image/jpg", "image/png", "image/gif"]}
-        minSize={1024}
-        maxSize={3072000}
+    <Container>
+      <Tabs
+        value={tab}
+        onChange={(e, val) => onTabChange(val)}
+        centered
+        style={{ paddingBottom: "30px", paddingTop: "15px" }}
       >
-        {({getRootProps, getInputProps}) => (
-          <section>
-            <CampaignDropzone {...getRootProps()}>
-              <input {...getInputProps()} />
-              {cover === null && <p>Drag 'n' drop your campaign cover image, or click to select an image</p>}
-              {cover !== null && <img src={cover} alt="campaign cover" />}
-            </CampaignDropzone>
-          </section>
-        )}
-      </Dropzone>
-      <CampaignTextField
-        label="Campaign Name"
-        variant="outlined"
-        required={!draft}
-        value={name}
-        onChange={(e) => setName(e.target.value)}
+        <Tab label="campaign" />
+        <Tab label="roles" />
+        <Tab label="review" />
+      </Tabs>
+      <CampaignTab
```

Comment:
hmm this is a bit messy, is it possible to extract these into one state? so the campaign object would be one state

Reply:
I'm not sure what you mean by this exactly D: